detox/src/utils/childProcess/exec.test.js

```diff
@@ -94,11 +94,14 @@ describe('exec', () => {
         retrying: true,
       },
     };
+
+    logger.debug.mockClear();
+
     await exec.execWithRetriesAndLogs('bin', options);
 
     expect(cpp.exec).toHaveBeenCalledWith(`bin --argument 123`, { timeout: 0 });
-    expect(logger.debug).toHaveBeenCalledWith({ event: 'EXEC_RETRY', retryNumber: 2 }, '(Retry #1)', 'bin --argument 123');
```

Comment:
Why did you decide to remove `retryNumber` metadata field?

Reply:
I figured that it's not very productive since the retry count appears in the log itself. WDYT?

Reply:
Ok.